test/functional/Java11andUp/src/org/openj9/test/condy/PrimitiveCondyTest.java

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+package org.openj9.test.condy;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.testng.log4testng.Logger;
+
+public class PrimitiveCondyTest {
+	/**
+	 * In all of the following test methods, methods from PrimitiveCondyMethods are called 3 times
```

Comment:
@DanHeidinga  I have commented in details why I chose methods to call multiple times (3).
Basically for testing JIT with resolved cases I need to make sure it is ran atleast one time in interpreted mode. Also
for unresolved cases (count=0), first invocation of the method will run into path where we are calling Bootstrapo methods which resolves condy and branch to call helper to resolve this is patched to NOP. 
Second invocation makes sure that once Condy is resolved by JIT helper, JIT loads correct value.